packages/framework/src/Framework/Features/Publications/Models/PublicationFieldType.php

```diff
@@ -91,26 +93,26 @@ public function getValidationRules(bool $reload = true): Collection
             case 'integer':
             case 'string':
             case 'text':
-                if ($doBetween) {
+                if ($useRange) {
                     $fieldRules->add("between:$this->min,$this->max");
                 }
                 break;
             case 'image':
                 $mediaFiles = PublicationService::getMediaForPubType($this->publicationType, $reload);
                 $valueList = $mediaFiles->implode(',');
-                $fieldRules->add("in:$valueList");
+                $fieldRules->add("in:$valueList"); // FIXME What if the list is empty?
```

Comment:
A few questions here:

1. What if the list is empty?
2. Now the items look like 'in:_media/foo/bar.jpg', but do we really need the directory information? Wouldn't it suffice with just 'in:bar.jpg' since we already know what directory it is in? We could then easily qualify it within the template and/or via a helper method.

Reply:
Okay so for the first one, I think it should be left blank as I assume that allows nothing to be selected, which should be the intended behaviour when there is nothing to select from.

For the second, I think the principle of least astonishment here resolves to keep it as it is now for clarity. At least for now.